`TestSnapshotCheckColsMySQL` test is failing

### Describe the bug
One of the functional tests is failing starting in dbt-mysql 1.2.0a1.

### Steps To Reproduce
```shell
pytest -v --profile mysql tests/functional/adapter/test_basic.py::TestSnapshotCheckColsMySQL::test_snapshot_check_cols
pytest -v --profile mysql5 tests/functional/adapter/test_basic.py::TestSnapshotCheckColsMySQL::test_snapshot_check_cols
pytest -v --profile mariadb tests/functional/adapter/test_basic.py::TestSnapshotCheckColsMySQL::test_snapshot_check_cols
```

### Screenshots and log output
```
FAILED tests/functional/adapter/test_basic.py::TestSnapshotCheckColsMySQL::test_snapshot_check_cols - AssertionError: dbt exit state did not match expected

Compilation Error in macro snapshot_check_all_get_existing_columns (macros/materializations/snapshot/snapshot.sql)
  macro 'dbt_macro__snapshot_check_all_get_existing_columns' takes not more than 2 argument(s)
  
  > in macro snapshot_check_strategy (macros/materializations/snapshots/strategies.sql)
  > called by macro materialization_snapshot_mysql (macros/materializations/snapshot/snapshot.sql)
  > called by macro snapshot_check_all_get_existing_columns (macros/materializations/snapshot/snapshot.sql)
```

### Additional context
Skipping this test until it is fixed.

--- a/dbt/include/mariadb/macros/materializations/snapshot/snapshot.sql
+++ b/dbt/include/mariadb/macros/materializations/snapshot/snapshot.sql
@@ -114,29 +114,6 @@
 
 {% endmaterialization %}
 
-{% macro snapshot_check_all_get_existing_columns(node, target_exists) -%}
-    {%- set query_columns = get_columns_in_query(node['compiled_sql']) -%}
-    {%- if not target_exists -%}
-        {# no table yet -> return whatever the query does #}
-        {{ return([false, query_columns]) }}
-    {%- endif -%}
-    {# handle any schema changes #}
-    {%- set target_table = node.get('alias', node.get('name')) -%}
-    {%- set target_relation = adapter.get_relation(database=None, schema=node.schema, identifier=target_table) -%}
-    {%- set existing_cols = get_columns_in_query('select * from ' ~ target_relation) -%}
-    {%- set ns = namespace() -%} {# handle for-loop scoping with a namespace #}
-    {%- set ns.column_added = false -%}
-
-    {%- set intersection = [] -%}
-    {%- for col in query_columns -%}
-        {%- if col in existing_cols -%}
-            {%- do intersection.append(col) -%}
-        {%- else -%}
-            {% set ns.column_added = true %}
-        {%- endif -%}
-    {%- endfor -%}
-    {{ return([ns.column_added, intersection]) }}
-{%- endmacro %}
 
 {% macro snapshot_staging_table(strategy, source_sql, target_relation) -%}
 

--- a/dbt/include/mysql/macros/materializations/snapshot/snapshot.sql
+++ b/dbt/include/mysql/macros/materializations/snapshot/snapshot.sql
@@ -113,27 +113,3 @@
   {{ return({'relations': [target_relation]}) }}
 
 {% endmaterialization %}
-
-{% macro snapshot_check_all_get_existing_columns(node, target_exists) -%}
-    {%- set query_columns = get_columns_in_query(node['compiled_sql']) -%}
-    {%- if not target_exists -%}
-        {# no table yet -> return whatever the query does #}
-        {{ return([false, query_columns]) }}
-    {%- endif -%}
-    {# handle any schema changes #}
-    {%- set target_table = node.get('alias', node.get('name')) -%}
-    {%- set target_relation = adapter.get_relation(database=None, schema=node.schema, identifier=target_table) -%}
-    {%- set existing_cols = get_columns_in_query('select * from ' ~ target_relation) -%}
-    {%- set ns = namespace() -%} {# handle for-loop scoping with a namespace #}
-    {%- set ns.column_added = false -%}
-
-    {%- set intersection = [] -%}
-    {%- for col in query_columns -%}
-        {%- if col in existing_cols -%}
-            {%- do intersection.append(col) -%}
-        {%- else -%}
-            {% set ns.column_added = true %}
-        {%- endif -%}
-    {%- endfor -%}
-    {{ return([ns.column_added, intersection]) }}
-{%- endmacro %}

--- a/dbt/include/mysql5/macros/materializations/snapshot/snapshot.sql
+++ b/dbt/include/mysql5/macros/materializations/snapshot/snapshot.sql
@@ -114,30 +114,6 @@
 
 {% endmaterialization %}
 
-{% macro snapshot_check_all_get_existing_columns(node, target_exists) -%}
-    {%- set query_columns = get_columns_in_query(node['compiled_sql']) -%}
-    {%- if not target_exists -%}
-        {# no table yet -> return whatever the query does #}
-        {{ return([false, query_columns]) }}
-    {%- endif -%}
-    {# handle any schema changes #}
-    {%- set target_table = node.get('alias', node.get('name')) -%}
-    {%- set target_relation = adapter.get_relation(database=None, schema=node.schema, identifier=target_table) -%}
-    {%- set existing_cols = get_columns_in_query('select * from ' ~ target_relation) -%}
-    {%- set ns = namespace() -%} {# handle for-loop scoping with a namespace #}
-    {%- set ns.column_added = false -%}
-
-    {%- set intersection = [] -%}
-    {%- for col in query_columns -%}
-        {%- if col in existing_cols -%}
-            {%- do intersection.append(col) -%}
-        {%- else -%}
-            {% set ns.column_added = true %}
-        {%- endif -%}
-    {%- endfor -%}
-    {{ return([ns.column_added, intersection]) }}
-{%- endmacro %}
-
 {% macro snapshot_staging_table(strategy, source_sql, target_relation) -%}
 
     select

--- a/tests/functional/adapter/test_basic.py
+++ b/tests/functional/adapter/test_basic.py
@@ -9,8 +9,12 @@ from dbt.tests.adapter.basic.test_empty import BaseEmpty
 from dbt.tests.adapter.basic.test_ephemeral import BaseEphemeral
 from dbt.tests.adapter.basic.test_incremental import BaseIncremental
 from dbt.tests.adapter.basic.test_generic_tests import BaseGenericTests
-from dbt.tests.adapter.basic.test_snapshot_check_cols import BaseSnapshotCheckCols
-from dbt.tests.adapter.basic.test_snapshot_timestamp import BaseSnapshotTimestamp
+from dbt.tests.adapter.basic.test_snapshot_check_cols import (
+    BaseSnapshotCheckCols,
+)
+from dbt.tests.adapter.basic.test_snapshot_timestamp import (
+    BaseSnapshotTimestamp,
+)
 from dbt.tests.adapter.basic.test_adapter_methods import BaseAdapterMethod
 from dbt.tests.util import run_dbt, check_relations_equal
 
@@ -47,8 +51,6 @@ class TestGenericTestsMySQL(BaseGenericTests):
     pass
 
 
-# TODO
-@pytest.mark.skip(reason="known failure to fix")
 class TestSnapshotCheckColsMySQL(BaseSnapshotCheckCols):
     pass
 

